modules/ossm-release-2-4-15.adoc

```diff
@@ -0,0 +1,26 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-4-15_{context}"]
+= {SMProductName} version 2.4.15
+
+This release of {SMProductName} is included with the {SMProductName} Operator 2.6.6 and is supported on {product-title} 4.14 and later. This release addresses Common Vulnerabilities and Exposures (CVEs).
```

Comment:
Please remove sentence "This release addresses Common Vulnerabilities and Exposures (CVEs)" for 2.4.15 we do not have CVE fixes. :+1: